desktop/app/index.js

```diff
@@ -81,11 +76,18 @@ function start () {
   app.once('ready', () => {
     mainWindow = MainWindow()
     storeHelper(mainWindow)
-    if (app.dock && !mainWindow.initiallyVisible) {
-      app.dock.hide()
+    if (!mainWindow.initiallyVisible) {
+      hideDockIcon()
     }
   })
 
+  // Called when the user clicks the dock icon
+  // We'll show the main window if there are no visible
+  // windows
+  app.on('activate', (e, hasVisibleWindows) => {
```

Comment:
Were you intending to use `hasVisibleWindows` here? Or can those params be deleted from the arrow function signature?